### PR TITLE
Add example for configuration object usage

### PR DESCRIPTION
**Pull request by Eskibear**

### CODE REVIEW

File: examples/configObject.mjs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as dotenv from "dotenv";
+dotenv.config()
+
+/**
+ * This example demostrates how to construct a configuration object from settings loaded from Azure App Configuration.
```
Comment (avanigupta) [inline examples/configObject.mjs:8]:
```suggestion
 * This example demonstrates how to construct a configuration object from settings loaded from Azure App Configuration.

```

File: examples/configObject.mjs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+// Licensed under the MIT license.
+
+import * as dotenv from "dotenv";
+dotenv.config()
+
+/**
+ * This example demostrates how to construct a configuration object from settings loaded from Azure App Configuration.
+ * If you are using configuration object instead of Map-styled settings, it would minimize the code changes required to use Azure App Configuration in your application.
+ *
+ * When you import configuration into Azure App Configuration from a local .json file, the keys are automatically flattened with specified separator (default is ".").
```
Comment (avanigupta) [inline examples/configObject.mjs:11]: >  (default is ".")

We dont have any default separator for import

**Review by avanigupta — APPROVED** [review]

File: examples/configObject.mjs (hunk 2 above)
Comment (Eskibear) [reply examples/configObject.mjs]: fixed in dc4578d

File: examples/configObject.mjs (hunk 1 above)
Comment (Eskibear) [reply examples/configObject.mjs]: The typo is fixed in dc4578d

**Pull request merged**